diff --git a/configure b/configure
index 17625e1..4382b85 100755
--- a/configure
+++ b/configure
@@ -5788,7 +5788,7 @@ else
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; }; then :
-  libc_cv_ssp=yes
+  libc_cv_ssp=no
 else
   libc_cv_ssp=no
 fi
@@ -5808,7 +5808,7 @@ else
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; }; then :
-  libc_cv_ssp_strong=yes
+  libc_cv_ssp_strong=no
 else
   libc_cv_ssp_strong=no
 fi
