Index: gcc/incpath.c
===================================================================
--- gcc/incpath.c	(revision 200117)
+++ gcc/incpath.c	(working copy)
@@ -475,6 +475,22 @@
 
   target_c_incpath.extra_includes (sysroot, iprefix, stdinc);
 
+  if (1) {
+    char *inc_path;
+
+    if (gcc_exec_prefix != NULL) {
+      inc_path = concat(gcc_exec_prefix, "../../include", NULL);
+    } else {
+      inc_path = concat(sysroot, "/include", NULL);
+    }
+
+    add_path (inc_path, AFTER, 1, false);
+
+    if (verbose)
+      fprintf(stderr, "\nsysroot: %s\niprefix: %s\ngcc_exec_prefix: %s\ninc_path: %s\ncpp_EXEC_PREFIX: %s\ncpp_PREFIX: %s\n\n",
+          sysroot, iprefix, gcc_exec_prefix, inc_path, cpp_EXEC_PREFIX, cpp_PREFIX);
+  }
+
   merge_include_chains (sysroot, pfile, verbose);
 
   cpp_set_include_chains (pfile, heads[QUOTE], heads[BRACKET],
