Index: libgcc/config.host
===================================================================
--- libgcc/config.host	(revision 200117)
+++ libgcc/config.host	(working copy)
@@ -581,13 +581,7 @@
 	    tmake_file="i386/t-mingw-pthread $tmake_file"
 	    ;;
 	esac
-	# This has to match the logic for DWARF2_UNWIND_INFO in gcc/config/i386/cygming.h
-	if test x$enable_sjlj_exceptions = xyes; then
-		tmake_eh_file="i386/t-sjlj-eh"
-	else
-		tmake_eh_file="i386/t-dw2-eh"
-		md_unwind_header=i386/w32-unwind.h
-	fi
+	tmake_eh_file="i386/t-sjlj-eh"
 	# Shared libgcc DLL install dir depends on cross/native build.
 	if test x${build} = x${host} ; then
 		tmake_dlldir_file="i386/t-dlldir"
@@ -605,12 +599,7 @@
 	    tmake_file="i386/t-mingw-pthread $tmake_file"
 	    ;;
 	esac
-	# This has to match the logic for DWARF2_UNWIND_INFO in gcc/config/i386/cygming.h
-	if test x$enable_sjlj_exceptions = xyes; then
-		tmake_eh_file="i386/t-sjlj-eh"
-	else
-		tmake_eh_file="i386/t-seh-eh"
-	fi
+	tmake_eh_file="i386/t-seh-eh"
 	# Shared libgcc DLL install dir depends on cross/native build.
 	if test x${build} = x${host} ; then
 		tmake_dlldir_file="i386/t-dlldir"
